packages/wallet/src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { Link } from '@/ui/Link'
+import { Logo } from '@/ui/Logo'
+
+type NavigationItemProps = {
+  name: string
+  href: string
+}
+
+const navigationItems: NavigationItemProps[] = [
+  {
+    name: 'Home',
+    href: '/'
+  },
+  {
+    name: 'Accounts',
+    href: '/accounts'
+  },
+  {
+    name: 'Developer',
+    href: '/developer'
+  },
+  {
+    name: 'Grants',
+    href: '/grants'
+  },
+  {
+    name: 'Settings',
+    href: '/settings'
+  }
+]
+
+const Sidebar = () => {
+  return (
+    <div className="hidden md:fixed md:inset-y-0 md:flex md:w-60 md:flex-col">
```

Comment:
We have 3 divs here that act as a container, couldn't we merge the divs, or the 2 divs with flex layout?